### PR TITLE
Update CoreClr/CoreFx and mark packages as stable for 1.1.7

### PR DESCRIPTION
**Pull request by wtgodbe**
@safern PTAL
@weshaggard I think Santi is OOF today, would you mind taking a look as well?

### CODE REVIEW

File: pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj [hunk 1]
```diff
@@ -76,7 +76,7 @@
       <Version>1.0.2</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.0.12-servicing-26117-01</Version>
```
Comment (safern) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: You need to update DotNetHostPolicy here as well to the 1.0.0 equivalent. I think it’s 1.0.11?

Comment (wtgodbe) [reply pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: Wouldn't it be 1.0.10? https://github.com/dotnet/core-setup/pull/3664/files#diff-c0a524ec6ca51973079c79a8e15af029R324

Comment (safern) [reply pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: Yes. 1.0.10

File: pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj [hunk 2]
```diff
@@ -70,13 +70,13 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.8</Version>
+      <Version>1.0.10</Version>
```
Comment (wtgodbe) [inline pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: @safern is there a reason this wasn't updated to 1.0.9 last time?

Comment (safern) [reply pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj]: I might have missed it.

**Pull request merged**